composer.json

```diff
@@ -21,15 +21,16 @@
         }
     },
     "require": {
```

Comment:
We should require php >= 7.2 here.

Reply:
I'm not sure it is necessary - it is implicated by other dependencies 🤔 . Maybe just for clarity, but it won't have any effect - the dependencies already require ^7.2.